cli/src/commands/account.rs

```diff
@@ -40,7 +40,14 @@ impl Account {
             Self::New { seed } => {
                 // Sample a new Aleo account.
                 let account = snarkos_account::Account::from(match seed {
-                    Some(seed) => PrivateKey::<Network>::new(&mut ChaChaRng::seed_from_u64(seed))?,
+                    Some(bigint) => {
+                        let mut bytes = bigint.to_bytes_be().1;
+                        bytes.truncate(32);
+                        bytes.resize(32, 0);
```

Comment:
This modifies the original value passed as an argument for numbers taking less amount of bytes than 32.
Since `Vec::resize` is padding with zeros at the end, eg. `[1]` will become `[1, 0, 0, 0, 0, 0, ...]`.

```rust
    let num = 1231275789u64;
    let mut bytes = num.to_be_bytes().to_vec();
    bytes.truncate(32);
    bytes.resize(32, 0);

    assert_eq!(
        num.to_bigint().unwrap(),
        BigInt::from_bytes_be(Sign::Plus, &bytes)
    );
```

```
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `1231275789`,
 right: `7728843391871504577583034288497874377335230004640899776989253074944`', src/main.rs:8:5
```

Reply:
`le` instead of `be` would probably work

Reply:
+1 on using `_le` to be consistent with the rest of our stack.